src/pykeen/evaluation/evaluator.py

```diff
@@ -700,6 +700,22 @@ def evaluate(
     return result
 
 
+def _predict_scores(
```

Comment:
should this just go in the model itself?

Reply:
f2a1cdd